client/sandbox/react-nextjs/pages/play/date-conversion.tsx

```diff
@@ -0,0 +1,141 @@
+'use client';
+import config from '../../config';
+import { init, i, id } from '@instantdb/react';
+import { useRouter } from 'next/router';
+import EphemeralAppPage from '../../components/EphemeralAppPage';
+import React, { useState } from 'react';
+import { useSearchParams } from 'next/navigation';
+
+const exampleTodos = [
+  'Water the plants',
+  'Feed the cat',
+  'Make the bed',
+  'Do the laundry',
+  'Wash the car',
+  'Buy the groceries',
+  'Clean the house',
+  'Do the dishes',
+];
+
+const getRandomTodo = () =>
+  exampleTodos[Math.floor(Math.random() * exampleTodos.length)];
+
+const dateTypes = {
+  number: new Date().getTime(),
+  string: new Date().toISOString(),
+  jsonString: new Date().toJSON(),
+  invalidJsonString: JSON.stringify(new Date()),
+  invalidString: 'invalid',
+  backendCanParseOnly: '2025-01-02T00:00:00-08',
```

Comment:
What does it mean to be `backendCanParseOnly`? Isn't this an example of something the client can coerce?